plugin/rocks.lua

```diff
@@ -6,6 +6,25 @@ local nio = require("nio")
 local adapter = require("rocks.adapter")
 local config = require("rocks.config.internal")
 
+-- Initialize the luarocks loader
+if config.enable_luarocks_loader then
+    local sc = vim.system({ config.luarocks_binary, "which", "luarocks.loader" }):wait()
+    local luarocks_path = sc.stdout and sc.stdout:match("(%S+)loader.lua")
```

Comment:
I assume we are doing the whole "luarocks which" check as we also permit users to use their system installation of luarocks, right?

Reply:
Yes.
I've just added a change that doesn't call `vim.system` if the luarocks path `<rocks_config>/bin/luarocks`.